src/app/services/speech_recognition.py

```diff
@@ -0,0 +1,92 @@
+import requests
+from typing import Union, Literal, Optional
+from dataclasses import dataclass
+
+from jinja2 import Environment, PackageLoader
+
+from app.services.base import Service, ContainerConfig
+from app.job import LocalJobConfig, SlurmJobConfig, EC2JobConfig
+from app.logger import logger
+
+
+# NOTE: TextGeneration container is optimized for NVIDIA A100, A10G and T4 GPUs with
+# CUDA 12.2+ and requires NVIDIA Container Toolkit on the service host. The image
+# was built to run on GPU and will not reliably work without GPU support.
+
+
+SpeechRecognitionModels = {
```

Comment:
Remove. This snuck into text generation service module due to an abandoned earlier idea. We can remove it there, too.